tests/utils.py

```diff
@@ -131,5 +131,13 @@ def mock_request_get(url, timeout):
                 {"kind": "contract", "contract": "tz1gtHbmBF3TSebsgJfJPvUB2e9x8EDeNm6V", "change": "14626175032"}
             ]
         })
+    if path == "/chains/main/blocks/178177/helpers/baking_rights":
```

Comment:
To which tz address does this path belong to so that we need to mock it? Does the real path also return an empty list? I think it might be helpful to have tests for accessing endorsing_rights and baking_rights.

Reply:
This block 178177 is actually never being queried during tests so I removed it. But the one below (196609) is needed during test of `cycle=51&delegate=tz1gtHbmBF3TSebsgJfJPvUB2e9x8EDeNm6V` in `TestCalculatePhases.test_process_payouts`.

The real path does not return an empty list. In this case, the model get filled with number of bake/endorses at zero, but it does not matter since we are not testing estimated rewards.